code_challenges.rb

```diff
@@ -46,6 +77,11 @@
 # Expected output: 100
 
 
+#def add_array
+#  let i=0
+#  i[0]+i[1]+i[2]
+#end
+#p add_array
```

Comment:
again, this method would only work with arrays with a length of 3. we need to be able to iterate through the array to add the values